Docs/Architecture/Simulation.md

```diff
@@ -2,35 +2,53 @@
 
 ## Simulation is the process of running our robot code on a computer other than the robot to test it without being limited by physical hardware
 
-Sometimes simulation includes physics simulation, where we use a mathematical model of a mechanism to predict how it will behave.
-Sometimes this will be very accurate.
-Sometimes it will make a lot of untrue assumptions about the mechanism.
-Both are useful, since we will likely have to retune the mechanism one way or another once we have real hardware.
-Deciding how much effort to put into a simulation model is a key part of the intuition you will build as you code more robots.
+Generally speaking, simulation depends on sending the inputs we would like to test to a model of a system and analyzing the simulated results.
+There are several ways to do this, depending on what you’re simulating.
+This could be something small, like simulated readings of an encoder, or something more complex, like a physics simulation of an arm.
+Simulation is important for a couple of reasons—we can test something without risking breaking an actual physical part, as well as being able to concurrently develop code as those physical components are being built (which is often the case during the build season).
 
 We have a number of tools at our disposal for simulation.
-One is WPILibs built in desktop simulation.
-This provides a way for us to send joystick inputs from a keyboard and run robot code on a computer.
+One is WPILib’s built in desktop simulation.
+This handles the process of sending inputs (from joysticks, widgets in the sim dashboard, etc) and running robot code on a computer.
 This is the backbone of our simulation.
 
-Then we have WPILibs simulation physics classes.
-These are built around a state-space model of a class of mechanism, and exist for many common mechanisms.
+The sim graphical user interface (GUI) looks like this and can be launched through the WPILib command palette.
+
+<img src="../../Assets/SimGUI.webp" alt="Screenshot of the sim GUI" width="600"/>
```

Comment:
I would recommend either explaining some of the important buttons/panels in the ui or explicitly handwaiving them